build/configure

```diff
@@ -130,7 +131,7 @@ fi
 
 (cd $scxpal_dir/build/ && chmod ug+x ./configure; ./configure $enable_debug $enable_bullseye $enable_ulinux_qual)
 
-omi_configure_quals="--enable-preexec ${enable_debug} ${enable_purify_agent} ${enable_purify_server} ${omi_configure_quals}"
```

Comment:
Yes, a bunch of flags (including all of the OpenSSL building) was moved to OMI. It's all wrapped up into `--enable-microsoft` for most of it, and `--enable-ulinux` for universal builds against multiple versions of SSL. Giving both (`--enable-microsoft --enable-ulinux`) does both, which is what our normal Linux builds will be doing.